tests/test_intercept.js

```diff
@@ -289,6 +289,20 @@ test('filter path with function', async t => {
   scope.done()
 })
 
+test('filter path with function on intercept', async t => {
```

Comment:
Is this a test for the transformPathFunction bug fix? Could you pull it into a new PR?